roles_n_rules/functions.py

```diff
@@ -11,6 +11,9 @@
 import management.db as db
 from main_classes import Mailbox
 
+def myrole():
```

Comment:
Don't you need to pass on the message as an input if you want this to work?

Reply:
Have I misunderstood the myrole() function? Otherwise, I thought all it needed was the message author unless you want me to make it so that it needs a user id but that is by default equal to the message author.